bcgov/biohubbc
# BHBC-2251: EML Project/Survey type

# Overview

## Links to Jira tickets

[BHBC-2251](https://quartech.atlassian.net/browse/BHBC-2251?atlOrigin=eyJpIjoiYTM5ZTE1ZTg3NmU4NGY3ODhhODBkYWMwMzQ0YTU3Y2UiLCJwIjoiaiJ9)

## Description of relevant changes

- EML generated during project submission will now be tagged with a project type 'PROJECT' or 'SURVEY'

## Testing Steps
To test fully you will need to be running Biohub Platform on the BHBC-2251 branch

- Create a project in SIMS
- Add a Survey to the Project
- Submit Project to Biohub
- Open Biohub and search for newly created Project/ Survey
- Open the details for Project page
- See project specific details as well as `Species Inventory Project` under the project title
- Open the details for Survey page
- See survey specific details as well as `Species Inventory Survey`


### Code


### Style


### Documentation


### Tests


### Linting/Formatting

       _See the `lint` commands in package.json_
       _See the `format` commands in package.json_

### SonarCloud
